geode-core/src/main/java/org/apache/geode/management/builder/ClusterManagementServiceBuilder.java

```diff
@@ -32,6 +32,8 @@ public static GeodeBuilder buildWithCache() {
       org.apache.geode.management.client.ClusterManagementServiceBuilder.Builder {
     GeodeBuilder setCredentials(String username, String password);
 
+    GeodeBuilder setAuthToken(String authToken);
```

Comment:
Some javadocs are needed here. It looks like if you set this to a non-null value then the username and password are ignored. It would be good to mention that in the javadocs.